spec/stud/task_spec.rb

```diff
@@ -12,5 +12,19 @@
       task = Stud::Task.new { raise Insist::Failure }
       insist { task.wait }.fails
     end
+
+    it "should support stop! and stop?" do
+      task = Stud::Task.new do
+        sleep(100)
+        Stud.stop?
+      end
+
+      # make sure the task enters the sleep method
+      sleep(1)
+
+      task.stop!
+      insist { task.stop? } == true
+      insist { task.wait } == true
```

Comment:
I won't in this PR, I prefer to keep consistency and a new PR should be created to refactor the specs using rspec. Also, don't forget this is not a logstash project per-se, we can always propose but ultimately it is up to @jordansissel to agree for this.